js/build_map.js

```diff
@@ -87,27 +88,28 @@ function create_map() {
 	stateData = topojson.feature(arguments[1], arguments[1].objects.states);
 	countyCentroids = topojson.feature(arguments[2], arguments[2].objects.foo);
 	
-  // relies on countyCentroids as global vars
-  pieformdata = pieData(countyCentroids);
-  console.log(pieformdata);
+  // also prepare the centroid data for presentation as pie charts
+  pieFormData = pieData(countyCentroids);
```

Comment:
I wonder if there's a way to have this start, but not necessarily need to finish before moving on. If I remember correctly, it is waiting for this line to finish before continuing, but if this isn't the initial view, that is unnecessary.

Reply:
oh, good idea! I think it's taking about 40 ms to run pieData, and yes, that should be optional/delayed if we're not going to use it right away. i suspect this change may become less important if and when we use pie slices to represent circles, but at least for now i'm happy to make this change.